client/stylesheets/_responsive.scss

```diff
@@ -403,6 +450,29 @@
     .profileCardLi {
         margin: 6px 5px !important;
     }
+
+    .profileChatBtn a, .optionsProClick ul .clicker {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `profileChatBtn` should be written in lowercase with hyphens<br>Selector `optionsProClick` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line